articles/support/matrix.md

```diff
@@ -0,0 +1,801 @@
+    <td>May receive bug fixes</td>
+  </tr>
+<table>
+
+### Lock
+
+<table class="table">
+  <tr>
+    <th>Library</th>
+    <th>Certified</th>
+    <th>Features</th>
+    <th>Bug Fixes</th>
+  </tr>
+  <tr>
+    <td>Lock.js</td>
+    <td>v10</td>
+    <td>v10</td>
+    <td>v9</td>
+  </tr>
+  <tr>
+    <td>Lock.swift</td>
+    <td>v2</td>
+    <td>v2</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Lock.iOS-macOS</td>
+    <td></td>
+    <td></td>
+    <td>v1</td>
+  </tr>
+  <tr>
+    <td>Lock.Android</td>
+    <td>v2</td>
+    <td>v2</td>
+    <td>v1</td>
+  </tr>
+</table>
+
+
+### Software Development Kits (SDKs)
+
+<table class="table">
+  <tr>
+    <th>SDK</th>
+    <th>Certified</th>
+    <th>Features</th>
+    <th>Bug Fixes</th>
+  </tr>
+  <tr>
+    <td>auth0.js</td>
+    <td>v7, v8</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0.Android</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0.Swift</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0.net</td>
+    <td></td>
+    <td>v3, v4</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0-java</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>node-auth0</td>
+    <td></td>
+    <td>v2</td>
+    <td></td>
+  </tr>
+</table>
+
+### Frameworks/Platform Integration SDKs
+
+<div class="alert alert-info">
+  Auth0 reserves the right to downgrade support for an SDK to Community-Supported at any time.
+</div>
+
+<table class="table">
+  <tr>
+    <th>SDK</th>
+    <th>Certified</th>
+    <th>Features</th>
+    <th>Community-Supported</th>
+  </tr>
+  <tr>
+    <td>angular-auth0</td>
+    <td>X</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>angular-lock</td>
+    <td>X</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>auth0-servlet</td>
+    <td></td>
+    <td></td>
+    <td>v3</td>
+  </tr>
+  <tr>
+    <td>auth0-spring-mvc</td>
+    <td></td>
+    <td></td>
+    <td>v1</td>
+  </tr>
+  <tr>
+    <td>auth0-spring-security-mvc</td>
+    <td></td>
+    <td></td>
+    <td>v1</td>
+  </tr>
+  <tr>
+    <td>auth0-spring-security-api</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>auth0-aspnet-owin</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>auth0-aspnet</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0.Windows.UWP</td>
+    <td></td>
+    <td></td>
+    <td>v0</td>
+  </tr>
+  <tr>
+    <td>Auth0.WinformsWPF</td>
+    <td></td>
+    <td></td>
+    <td>v0</td>
+  </tr>
+  <tr>
+    <td>auth0-oidc-client-net</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+</table>
+
+## Quickstarts and Samples
+
+### Mobile/Native Applications
+
+<table class="table">
+  <tr>
+    <th></th>
+    <th>Quickstart Provided + Auth0 Support</th>
+    <th>Bug Fixes Provided</th>
+    <th>Community Support Only</th>
+  </tr>
+  <tr>
+    <td>[Android](/quickstart/native/android)</td>
```

Comment:
Since we are using html we should add an anchor instead of the markdown link

```html
<td><a href="/quickstart/native/android">Android</a></td>
```